From e07e6642633a963c6c81ec63ed546910472612de Mon Sep 17 00:00:00 2001
From: Mikhail Anikin <mikhail.anikin@solid-run.com>
Date: Thu, 13 Jul 2023 18:03:13 +0300
Subject: [PATCH] RZ/G2LC som: sd card is non-removable

Right now we have two options: system can boot from emmc (wich is non-removable) and sd (wich is removable).
But with sd boot removing doesn't make any sence since sd contains bootloaders and filesystem.
Other scenarios are rare. This might be a temporary solution.
---
 arch/arm64/boot/dts/renesas/rzg2lc-sr-som.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/rzg2lc-sr-som.dtsi b/arch/arm64/boot/dts/renesas/rzg2lc-sr-som.dtsi
index 43658a0b3007..e4de0542a0d0 100644
--- a/arch/arm64/boot/dts/renesas/rzg2lc-sr-som.dtsi
+++ b/arch/arm64/boot/dts/renesas/rzg2lc-sr-som.dtsi
@@ -318,7 +318,8 @@ &sdhi0 {
 	vmmc-supply = <&reg_3p3v>;
 	vqmmc-supply = <&vccq_sdhi0>;
 	bus-width = <4>;
-  max-frequency = <50000000>;
+	non-removable;
+  	max-frequency = <50000000>;
 	status = "okay";
 };
 #endif
-- 
2.41.0

